packages/core/admin/admin/src/content-manager/components/BlocksEditor/Components.js

```diff
@@ -0,0 +1,96 @@
+import { Typography } from '@strapi/design-system';
+import styled, { css } from 'styled-components';
+
+export const H1 = styled(Typography).attrs({ as: 'h1' })`
+  font-size: ${42 / 16}rem;
+  line-height: ${({ theme }) => theme.lineHeights[1]};
+`;
+
+export const H2 = styled(Typography).attrs({ as: 'h2' })`
+  font-size: ${35 / 16}rem;
+  line-height: ${({ theme }) => theme.lineHeights[1]};
+`;
+
+export const H3 = styled(Typography).attrs({ as: 'h3' })`
+  font-size: ${29 / 16}rem;
+  line-height: ${({ theme }) => theme.lineHeights[1]};
+`;
+
+export const H4 = styled(Typography).attrs({ as: 'h4' })`
+  font-size: ${24 / 16}rem;
+  line-height: ${({ theme }) => theme.lineHeights[1]};
+`;
+
+export const H5 = styled(Typography).attrs({ as: 'h5' })`
+  font-size: ${20 / 16}rem;
+  line-height: ${({ theme }) => theme.lineHeights[1]};
+`;
+
+export const H6 = styled(Typography).attrs({ as: 'h6' })`
+  font-size: 1rem;
+  line-height: ${({ theme }) => theme.lineHeights[1]};
+`;
```

Comment:
These font-sizes match the designs but not the [fontSizes from the design system](https://github.com/strapi/design-system/blob/13ccb7766d6751703fa993dc95be84b4de1ee598/packages/strapi-design-system/src/themes/common-theme.ts#L30). @lucasboilly just want to confirm that is intentional?

Reply:
I believe it's intentional because we need 6 heading levels and the DS doesn't have that many

Reply:
```
[`${11 / 16}rem`, `${12 / 16}rem`, `${14 / 16}rem`, '1rem', `${18 / 16}rem`, `${32 / 16}rem`]
```

oh right got it here the smallest starts at 1rem 👍. My bad I just saw 6 font sizes and assumed they were all headings 🤦‍♂️ . thanks